### PR TITLE
Add Plugin HiToKoTo

### PR DESCRIPTION
**Pull request by MRNOBODY-ZST**

### CODE REVIEW

**Comment by AnzhiZhang** [comment]
Tag is wrong, it should be `information`.

File: plugins/hitokoto/plugin_info.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "id": "hitokoto",
+    "authors": [
+        {
+            "name": "MRNOBODY-ZST",
+            "link": "https://github.com/MRNOBODY-ZST"
+        }
+    ],
+    "repository": "https://github.com/MRNOBODY-ZST/HiToKoTo",
+    "branch": "main",
+    "labels": ["tool"],
```
Comment (AnzhiZhang) [inline plugins/hitokoto/plugin_info.json:11]: Label is wrong, it should be information.

**Review by AnzhiZhang — CHANGES_REQUESTED** [review]

**Review by AnzhiZhang — APPROVED** [review]

**Review by alex3236 — CHANGES_REQUESTED** [review]
- Move English description to English README or just remove it.
- Add a requirements.txt in your repo to show your requirement`requests`

**Review by AnzhiZhang — CHANGES_REQUESTED** [review]
If you do not need introduction, please remove README.md

**Pull request closed without merging**